mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -89,9 +89,7 @@ impl<S> StoreMqttEventHandler<S> {
     }
 
     fn update_unsubscribed(&mut self, sub: &str) {
-        if self.topic_mappers_updates.contains_key(sub) {
-            self.topic_mappers.remove(sub);
-        } else {
+        if self.topic_mappers.remove(sub).is_none() {
```

Comment:
because it is removing the update when the unsubscribe event was sent and by the time it got the ack it was already not there and not removing from topic_mappers